src/App.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
+
+import { Route } from 'react-router-dom';
+
 import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import { Navigation } from './components';
+import Home from './components/Home';
+import About from './components/About';
+import Contact from './components/Contact';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path="/" exact component={Home} />
+    <Route path="/About" exact component={About} />
+    <Route path="/Contact" exact component={Contact} />
+    
```

Comment:
It doesn't hurt to have exact on every Route, I think it's better to only have it on routes the need it. In this case the `path="/"` is the only one that needs it to stop it from showing up on `/about` and `/home`